docs/modules/reference/pages/glossary.adoc

```diff
@@ -83,7 +83,8 @@ HELM works with the Grafana analytics program to customize the data you can view
 Not to be confused with the https://helm.sh/[Kubernetes Helm] package manager.
 (See https://docs.opennms.com/helm/latest[HELM Documentation].)
 
-Horizon:: An open-source solution from OpenNMS that helps users visualize and monitor everything on their local and remote networks.
+Horizon:: An open-source solution from The OpenNMS Group that helps users visualize and monitor everything on their local and remote networks.
+(See https://www.opennms.com/horizon/[OpenNMS Horizon].)
```

Comment:
Let's remove the reference here and make the header the link. Could you add a similar link to the Meridian entry, but to https://www.opennms.com/meridian/ ?